pygmt/src/grdhisteq.py

```diff
@@ -63,7 +63,7 @@ class grdhisteq:  # pylint: disable=invalid-name
         h="header",
     )
     @kwargs_to_strings(R="sequence")
-    def _grdhisteq(grid, output_type, **kwargs):
+    def _grdhisteq(caller, grid, output_type, **kwargs):
```

Comment:
This one is a little complicated because it can output either a table or a grid.